lib/screens/app/wallet/transfer_form.dart

```diff
@@ -207,30 +204,44 @@ class _TransferFormState extends State<TransferForm>
-                    controller: controller,
-                    labelText: 'Transfer amount'.i18n,
-                    endText: 'SEEDS',
-                    autofocus: true,
-                    validator: (val) {
-                      String error;
-                      double availableBalance =
-                          double.tryParse(balance.replaceFirst(' SEEDS', ''));
-                      double transferAmount = double.tryParse(val);
-
-                      if (transferAmount == 0.0) {
-                        error = "Transfer amount cannot be 0.".i18n;
-                      } else if (transferAmount == null ||
-                          availableBalance == null) {
-                        error = "Transfer amount is not valid.".i18n;
-                      } else if (transferAmount > availableBalance) {
-                        error =
-                            "Transfer amount cannot be greater than availabe balance.".i18n;
-                      }
-                      return error;
-                    },
-                  ),
+                      keyboardType: TextInputType.numberWithOptions(signed: false, decimal: true),
+                      controller: controller,
+                      labelText: 'Transfer amount'.i18n,
+                      endText: 'SEEDS',
+                      autofocus: true,
+                      validator: (val) {
+                        String error;
+                        double availableBalance = double.tryParse(balance.replaceFirst(' SEEDS', ''));
+                        double transferAmount = double.tryParse(val);
+
+                        if (transferAmount == 0.0) {
+                          error = "Transfer amount cannot be 0.".i18n;
+                        } else if (transferAmount == null || availableBalance == null) {
+                          error = "Transfer amount is not valid.".i18n;
+                        } else if (transferAmount > availableBalance) {
+                          error = "Transfer amount cannot be greater than availabe balance.".i18n;
+                        }
+                        return error;
+                      },
+                      onChanged: (controller) {
+                        _setUSD(controller);
+                      }),
+                  Align(
+                      alignment: Alignment.topLeft,
+                      child: Padding(
+                          padding: EdgeInsets.fromLTRB(16, 5, 0, 0),
+                          child: Consumer<RateNotifier>(
+                            builder: (context, rateModel, child) {
+                              return Text(
+                                rateModel.rate == null
+                                    ? ""
+                                    : rateModel.rate.error
+                                        ? "Exchange rate load error".i18n
+                                        : '${rateModel.rate?.usdString(valueInUSD)}',
```

Comment:
`rateModel.rate` cant be null anymore since you checked for null 
`?` is no longer needed